### ISSUE TITLE
Can't install release 1.4. because of invalid setup file

### ISSUE THREAD

**Issue by bensteUEM** [bensteUEM/ChurchToolsAPI#70]
```
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-wnsgar97/churchtoolsapi_d9bee8eba01642e9887440cfedb664b7/setup.py", line 7, in <module>
          with open('docker/requirements.txt') as f:
      FileNotFoundError: [Errno 2] No such file or directory: 'docker/requirements.txt'
      [end of output]
```

**Comment by bensteUEM** [comment]
issue still exists when running
 pip install git+https://github.com/bensteUEM/ChurchToolsAPI.git@1.4.0#egg=ChurchToolsAPI
on venv with python 3.11 used in ChurchWebHelper Project

**Comment by bensteUEM** [comment]
Is already resolved with version 1.4.1
